build.sbt

```diff
@@ -29,7 +26,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "3.1." + patchVersion,
+  version := "3.2." + patchVersion,
```

Comment:
revert would be to go back to 3.0, but we don't do parallel branches, only go forward. And the revert is equally big to the original PR to be worth a version change